DuggaSys/resulted.js

```diff
@@ -770,84 +841,82 @@ function renderCell(col,celldata,cellid) {
-				str += "/>";
+		if (celldata.ishere === true || celldata.kind == 4) {
+			str += "<div class='gradeContainer resultTableText'>";
+			if (celldata.grade === null) {
+				str += makeSelect(celldata.gradeSystem, querystring['cid'], celldata.vers, celldata.lid, celldata.uid, celldata.grade, 'I', celldata.qvariant, celldata.quizId);
+			} else if (celldata.grade === -1) {
+				str += makeSelect(celldata.gradeSystem, querystring['cid'], celldata.vers, celldata.lid, celldata.uid, celldata.grade, 'IFeedback', celldata.qvariant, celldata.quizId);
+			} else {
+				str += makeSelect(celldata.gradeSystem, querystring['cid'], celldata.vers, celldata.lid, celldata.uid, celldata.grade, 'U', celldata.qvariant, celldata.quizId);
+			}
+			str += "<img id='korf' class='fist";
+			if (celldata.userAnswer === null && !(celldata.quizfile == "feedback_dugga")) { // Always shows fist. Should be re-evaluated
+				str += " grading-hidden";
+			}
+			str += "' src='../Shared/icons/FistV.png' onclick='clickResult(\"" + querystring['cid'] + "\",\"" + celldata.vers + "\",\"" + celldata.lid + "\",\"" + celldata.firstname + "\",\"" + celldata.lastname + "\",\"" + celldata.uid + "\",\"" + celldata.submitted + "\",\"" + celldata.marked + "\",\"" + celldata.grade + "\",\"" + celldata.gradeSystem + "\",\"" + celldata.lid + "\",\"" + celldata.qvariant + "\",\"" + celldata.quizId + "\");'";
+			str += "/>";
 			str += "</div>";
 
 			// Print submitted time and change color to red if passed deadline
 			str += "<div class='text-center resultTableText'";
-				for (var p = 0; p < moments.length; p++){
-					if (moments[p].link == celldata.quizId){
-						if (Date.parse(moments[p].deadline) < Date.parse(celldata.submitted)){
-							str += " style='color:red;'";
-						}
-						break;
+			for (var p = 0; p < moments.length; p++) {
+				if (moments[p].link == celldata.quizId) {
+					if (Date.parse(moments[p].deadline) < Date.parse(celldata.submitted)) {
+						str += " style='color:red;'";
 					}
+					break;
 				}
-				str += ">";
-				if (celldata.submitted.getTime() !== timeZero.getTime()){
-					str += celldata.submitted.toLocaleDateString()+ " " + celldata.submitted.toLocaleTimeString();
-				}
+			}
+			str += ">";
+			if (celldata.submitted.getTime() !== timeZero.getTime()) {
+				str += celldata.submitted.toLocaleDateString() + " " + celldata.submitted.toLocaleTimeString();
+			}
 			str += "</div>";
 
 			// Print times graded
 			str += "<div class='text-center resultTableText'>";
-				if(celldata.ishere===true && celldata.timesGraded!==0){
-					str += "Times Graded: " + celldata.timesGraded;
-				}
+			if (celldata.ishere === true && celldata.timesGraded !== 0) {
+				str += "Times Graded: " + celldata.timesGraded;
+			}
 			str += "</div>";
 		}
 		str += "</div>";
 		return str;
 	}
-
-  // When Filtering is activated then this "hides" all other data than what is specified.
-  else {
```

Comment:
Code segment with syntax error (lines 842-850)

Reply:
This is supposed to be fixed plus fixing the issue(adding a feature) that caused it in this branch https://github.com/HGustavs/LenaSYS/tree/TempBranchResultMerge.
Its the one that went bad during the big merge. GruppB already merged this into their branch so I'd suggest as a fix for your issue that you instead create a new pull request from this branch so we at gruppC also get the same fix + some result table functionality to hopefully minimize conflicts once we do the next merge